__testing__/apiEnrolled.test.js

```diff
@@ -0,0 +1,71 @@
+import 'regenerator-runtime/runtime';
+const mongoose = require('mongoose');
+const app = require('../server/index');
+const supertest = require('supertest');
+const request = supertest(app);
+const Enrolled = require('../db/enrolled.model');
+
+
+// connect to db prior to tests
+beforeAll(async () => {
+  const MONGO_URI = 'mongodb://localhost/testEnrolledDB';
+  await mongoose.connect(MONGO_URI, { useUnifiedTopology: true, useNewUrlParser: true });
+})
+
+// POST
+it('Should POST enrolled number to database', async (done) => {
+  const postedEnrolledNum = '18789';
+
+  const res = await request.post('/postEnrolled')
+    .send({
+      id: '1',
+      enrolled: postedEnrolledNum,
+    });
+
+  expect(res.status).toBe(200);
```

Comment:
You are testing all the successes (which is good), but do you also want to test failures / make sure the proper error status is returned when it's expected?